src/app/templates/index.html

```diff
@@ -1,11 +1,21 @@
 {% extends 'base.html' %}
 {% block left %}
+<style>
+    #counter-container {
+        margin-top: .50rem;
+    }
+    #counter {
+        padding: .25rem .75rem;
+    }
+</style>
```

Comment:
Nit-picking: Can this be moved to [where other styles are defined in base.html](https://github.com/nolanbconaway/sqlfluff-online/pull/7/files#diff-e0fc3e8c1e55368563a801f2ef793369R101-R105)?